lib/commons/text/unicode.js

```diff
@@ -121,7 +121,7 @@ function getPunctuationRegExp() {
 	 * Reference: https://en.wikipedia.org/wiki/Supplemental_Punctuation
 	 * -> \u2E00-\u2E7F Reference
 	 */
-	return /[\u2000-\u206F\u2E00-\u2E7F\\'!"#$%&()*+,\-.\/:;<=>?@\[\]^_`{|}~]/g;
+	return /[\u2000-\u206F\u2E00-\u2E7F\\'!"#$%&£¢¥§€()*+,\-.\/:;<=>?@\[\]^_`{|}~±]/g;
```

Comment:
Why are money signs considered punctuation? They technically fall under the BMP range for Currency Symbols `\u20A0-\u20CF` (covered in `getUnicodeNonBmpRegExp`)

Reply:
Created https://github.com/dequelabs/axe-core/issues/2128 to look into this later